src/client/project/__init__.py

```diff
@@ -114,47 +115,48 @@ class Project(BaseTolokaObject):
 
     @unique
     class AssignmentsIssuingType(ExtendableStrEnum):
-        """How to assign tasks:
+        """Settings for assigning tasks.
 
         Attributes:
-            AUTOMATED: A Toloker is assigned a task suite from the pool. You can configure the order
-                for assigning task suites.
-            MAP_SELECTOR: A Toloker chooses a task suite on the map. If you are using MAP_SELECTOR,
-                specify the text to display in the map by setting assignments_issuing_view_config.
+            AUTOMATED: A Toloker is assigned a task suite from a pool. You can configure the order
+                of assigning task suites.
+            MAP_SELECTOR: A Toloker chooses a task suite on the map.
+                A task view on the map is configured with the `Project.assignments_issuing_view_config` parameters.
         """
 
         AUTOMATED = 'AUTOMATED'
         MAP_SELECTOR = 'MAP_SELECTOR'
 
     @unique
     class ProjectStatus(ExtendableStrEnum):
-        """Project status:
+        """A project status.
 
         Attributes:
-            ACTIVE: A project is active
-            ARCHIVED: A project is archived
+            ACTIVE: The project is active.
+            ARCHIVED: The project is archived.
         """
 
         ACTIVE = 'ACTIVE'
         ARCHIVED = 'ARCHIVED'
 
     class AssignmentsIssuingViewConfig(BaseTolokaObject):
-        """How the task will be displayed on the map
+        """Task view on the map.
 
-        Used only then assignments_issuing_type == MAP_SELECTOR
+        These parameters are used when `Project.assignments_issuing_type` is set to `MAP_SELECTOR`.
 
         Attributes:
-            title_template: Name of the task. Tolokers will see it in the task preview mode.
-            description_template: Brief description of the task. Tolokers will see it in the task preview mode.
+            title_template: The name of a task. Tolokers see it in the task preview mode.
+            description_template: The brief description of a task. Tolokers see it in the task preview mode.
+            map_provider: A map provider.
         """
 
         @unique
         class MapProvider(ExtendableStrEnum):
-            """Map provider for assignments_issuing_view_config:
+            """A map provider.
 
             Attributes:
-                YANDEX: Use Yandex Maps as a map provider
-                GOOGLE: Use Google Maps as a map provider
+                YANDEX: Yandex Maps.
```

Comment:
Is it possible to reverse the order and make it alphabetic (first `GOOGLE`, then `YANDEX`)?